agent/config/builder.go

```diff
@@ -916,6 +916,7 @@ func (b *Builder) Build() (rt RuntimeConfig, err error) {
 		ConnectMeshGatewayWANFederationEnabled: connectMeshGatewayWANFederationEnabled,
 		ConnectSidecarMinPort:                  sidecarMinPort,
 		ConnectSidecarMaxPort:                  sidecarMaxPort,
+		ConnectTestCALeafRootChangeSpread:      b.durationVal("connect.test_ca_leaf_root_change_spread", c.Connect.TestCALeafRootChangeSpread),
```

Comment:
Put into the `Config` and map into the `RuntimeConfig` to simplify how test overrides are done. We now just inject a source after the tail that can include this value along with other things such as the `check_reap_interval`